_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: otel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# otel metrics raw
+
+The `otel metrics raw` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
+
+## Usage
+
+See the following `.yaml` configuration file:
```

Comment:
What YAML file? What is it called?

Reply:
This also refers to `pipeline.yaml` file. The code block that follows gives an example, which shows only the processor configuration section of a pipeline yaml.

Reply:
@asifsmohammed All other docs related to this one contain the following text under the Usage section:
```o get started, create the following `pipeline.yaml` file: 

```yaml
pipeline:
  source:
    file:
      path: "/full/path/to/logs_json.log"
      record_type: "event"
      format: "json"
  processor:
    - substitute_string:
        entries:
          - source: "message"
            from: ":"
            to: "-"
  sink:
    - stdout:
```
{% include copy.html %}```

Does this text apply here? Otherwise the Usage section is very short and doesn't provide much context.